docs/installation-guide/quickstart.rst

```diff
@@ -173,39 +189,40 @@ twice to have 3 masters (bootstrap + 2 new master).
 
 .. code-block:: shell
 
-   $ kubectl apply -f new-master-node.yaml
-   node/new-master-node created
+   $ kubectl apply -f new-control-plane-node.yaml
+   node/new-cp-node created
 
-4. Check that the new master node was added to the cluster.
+4. Check that the new control-plane node was added to the cluster.
 
 .. code-block:: shell
 
-   $ kubectl get nodes
+   $ kubectl get nodes --kubeconfig /etc/kubernetes/admin.conf
    NAME                   STATUS    ROLES                         AGE       VERSION
    bootstrap              Ready     bootstrap,etcd,infra,master   12d       v1.11.7
-   new-master-node        Unknown   etcd,master                   29s
+   new-cp-node            Unknown   etcd,master                   29s
 
-5. The new master node now need to be installed to change its status from
-   Unknown to Ready. You go into the master-bootstrap pod ...
+5. The new control-plane node now need to be installed to change its
+   status from Unknown to Ready. Obtaining a shell into the
+   salt-master-bootstrap pod ...
 
 .. code-block:: shell
 
-   $ kubectl exec salt-master-bootstrap -n kube-system -c salt-master -it bash
+   $ kubectl exec salt-master-bootstrap -n kube-system -c salt-master -it bash --kubeconfig /etc/kubernetes/admin.conf
```

Comment:
So I understand the name of this salt master pod, depends on the hostname ? in my case salt-master-bootstrap.novalocal

Reply:
Yes, that part is confusing - I'll add a note about this (static Pods have a specific suffix, I need to check if it's the hostname or the Node name)